src/pages/Home.tsx

```diff
@@ -1,166 +1,121 @@
+import PeopleGridDesktopPath from "../images/people_grid_desktop.png";
 
-const Hero = styled.img`
-  min-height: 26vh;
-  max-height: 22rem;
-  width: 100%;
-  object-fit: cover;
-  object-position: right;
+const HeroSection = styled.div`
+  background-color: #dbf0f9;
 `;
 
-const OverlaySection = styled.div`
-  margin-top: -8vh;
+const PeopleGridDesktop = styled.div`
+  @media (min-width: 1024px) {
+    background-image: url(${PeopleGridDesktopPath});
+    background-position: right bottom;
+    background-repeat: no-repeat;
+    background-size: contain;
+  }
 `;
 
-const HorizontalLineText = styled.div`
-  display: flex;
-  flex-direction: row;
-  margin: 2rem;
-  &:before,
-  &:after {
-    content: "";
-    flex: 1 1;
-    border-bottom: 1px solid;
-    margin: auto;
-    margin-right: 1rem;
-    margin-left: 1rem;
-  }
+const PeopleGridMobile = styled.img`
+  height: 5rem;
+  object-fit: cover;
 `;
 
-const VerticalLineText = styled.div`
-  display: flex;
-  flex-direction: column;
-  margin-right: 2rem;
-  margin-left: 2rem;
-  &:before,
-  &:after {
-    content: "";
-    flex: 1 1;
-    border-left: 1px solid;
-    margin: auto;
-    margin-bottom: 1rem;
-    margin-top: 1rem;
-  }
+const GuidedSearchSection = styled.div`
+  background-color: #183647;
+`;
+
+const ResourcesSection = styled.div`
+  background-color: #f5f9fc;
 `;
 
 function Home() {
   const { t } = useTranslation();
 
+  const [showCrisisAlert, setShowCrisisAlert] = useState(true);
+
   useEffect(() => {
     window.scrollTo(0, 0);
     logPageView();
   }, []);
 
   return (
     <>
-      <div className="position-relative padding-x-0 tablet:padding-x-2">
-        <Hero src={heroPath} alt="hero image" />
-      </div>
-      <GridContainer>
-        <OverlaySection>
+      <HeroSection className="padding-bottom-05">
+        <GridContainer>
           <Grid row>
-            <Grid col={12} desktop={{ col: 7 }} tablet={{ col: 8 }}>
-              <div
-                className="display-none tablet:display-block position-absolute bottom-0 left-0"
-                aria-hidden
-              >
-                <h1 className="text-white padding-2">{t("homePageHeading")}</h1>
-              </div>
-              <div className="tablet:display-none bg-white radius-lg padding-x-3 padding-top-3">
-                <h1 className="margin-0">{t("homePageHeading")}</h1>
-              </div>
-            </Grid>
             <Grid col={12}>
-              <div className="display-none tablet:display-block" aria-hidden>
-                <div className="radius-lg bg-white padding-4 display-flex desktop:padding-y-4 tablet:margin-x-2 desktop:margin-x-0">
-                  <ZipCard id="desktop_zip" />
-                  <VerticalLineText>{t("or")}</VerticalLineText>
-                  <GuidedSearchCard isMobile={false} />
-                </div>
+              <div className="padding-y-2">
+                {showCrisisAlert && (
```

Comment:
previously the spec was "show this unless 1) user has closed it 2) user is in guided search flow" just confirming 2 no longer desired

Reply:
yep - discussed w adam and we're ok w keeping this on just the homepage